formal-spec/introduction.tex

```diff
@@ -0,0 +1,23 @@
+\section{Introduction}
+\label{sec:introduction}
+
+
+\subsection{Scope}
+\label{sec:scope}
+
+In this specification we define an update protocol that covers the different
+phases of the software update process for blockchain systems. The following
+aspects are outside the scope of the current specification:
+
+\begin{description}
+\item[voting incentives] we assume that honest parties will participate in the
+  voting process. In reality an incentive mechanism might be needed for ensuring
```

Comment:
"..honest parties will participate in the  voting process." 
->
"..honest parties will participate in the  voting process. Also we use default delegation as a means to ensure participation in the voting process. In reality ..."